### PR TITLE
Fix torchtext imports

### PR DESCRIPTION
**Pull request by nikithamalgifb**
Fixes issue #259

### CODE REVIEW

**Comment by nikithamalgifb** [comment]
The torchtext imports haven't been updated yet for torchtext installed via: https://download.pytorch.org/whl/nightly/cu110/torch_nightly.html

So these changes are irrelevant for now in Master. Once, this nightly whl is updated, we can then merge this PR.

File: torchbenchmark/models/attention_is_all_you_need_pytorch/__init__.py [hunk 1]
```diff
@@ -7,8 +7,14 @@
 import torch
 import torch.nn.functional as F
 import torch.optim as optim
-from torchtext.data import Field, Dataset, BucketIterator
-from torchtext.datasets import TranslationDataset
+import torchtext.data
+from torchtext.data import Dataset, BucketIterator
+if torchtext.__version__ >= "0.9":
```
Comment (wconstab) [inline torchbenchmark/models/attention_is_all_you_need_pytorch/__init__.py:12]: does it work to say 'if torchtext.__version__' 
without first importing torchtext?

**Comment by wconstab** [comment]
@xuzhao9 can you help with the questions around cuda version and torchtext wheel?  seems there is a difference in torchtext version available for 10.2 and 11 cuda.  and maybe our CI config is not all using the consistent cuda version yet.

**Comment by Krovatkin** [comment]
@wconstab the script to build torchbench against custom pytorch also breaks because of this.

File: torchbenchmark/models/attention_is_all_you_need_pytorch/__init__.py (hunk 1 above)
Comment (nikithamalgifb) [reply torchbenchmark/models/attention_is_all_you_need_pytorch/__init__.py:12]: Tried it, it doesn't work.

**Comment by xuzhao9** [comment]
@wconstab @nikithamalgifb 
 Our CI is running only on 0.1 branch and CUDA 10.2. To me the current commit looks fine since it tries to import the latest packages first, and fallback to the old import if unavailable.

**Review by wconstab — APPROVED** [review]

**Pull request merged**